### ISSUE TITLE
Force Mail App to display all incoming mails in plain text

### ISSUE THREAD

**Issue by himpierre** [nextcloud/mail#4714]
#### Summary

Currently there seems to be no way to force the app to show incoming mails in plain text instead of html formated. That would be a high appreciated feature.

cheers, t.

**Comment by himpierre** [comment]
Yeah. Roundcube has a switch to turn display of html in mails on and off. I would love to have that in nextcloud mail.

cheers, t.

**Comment by gsantner** [comment]
Would be cool if it can be globally set to "deault to plaintext", and there is a button in email view to "display html". What all options gnome-evolution has, would be even better.

**Comment by gsantner** [comment]
Would be cool if it can be globally set to "deault to plaintext", and there is a button in email view to "display html".   
Even better if all said display options from gnome-evolution are added.

**Comment by ChristophWurst** [comment]
> Emails are flooded with tracking pixels and other call-home stuff.

FYI we filter them out. Anything external is blocked by default.

> Defaulting to html display, and not even having a plaintext display option is a no-go.

I'm happy to accept code that adds an action where you can toggle the mode if a message is sent as multipart.

**Comment by gsantner** [comment]
@ChristophWurst 

> FYI we filter them out. Anything external is blocked by default.

OK I see, and sometimes seems to work well, but seemingly often fails to block.
Here I browse mails on a spam-email-account with the default NextCloud Mail configuration + images blocked:  


https://user-images.githubusercontent.com/6735650/121394066-43164900-c951-11eb-9b28-f6c61da6d870.mp4

# Notes
I assure that I do not have running something that would produce requests to 99% of the domains shown in the log. For 100% I can say that I have not been playing LoL/RiotGames in years, never bought something from Otto-de. So these request cannot come any other software running in background on the PC ... but from `something of nextcloud`.

I don't really care what NextCloud, NextCloud Mail or my browser due the displayed content tries to render. Just looking at the DNS requests tells me enough that it tries to request `something` from there, whatever it is.

Please let me know if more information needed.  
Or is this working as expected? Sorry if I missed something obvious

# Some domains from log
![Screenshot_20210609-183518](https://user-images.githubusercontent.com/6735650/121395023-4231e700-c952-11eb-8018-e3f5fa48c52b.png)
![Screenshot_20210609-183644](https://user-images.githubusercontent.com/6735650/121395028-43631400-c952-11eb-8b31-b7bb1624ada4.png)
![Screenshot_20210609-183658](https://user-images.githubusercontent.com/6735650/121395032-43fbaa80-c952-11eb-83dc-3c4a5153d75c.png)
![Screenshot_20210609-183710](https://user-images.githubusercontent.com/6735650/121395033-44944100-c952-11eb-9fae-8bafb54a33df.png)
![Screenshot_20210609-183718](https://user-images.githubusercontent.com/6735650/121395036-452cd780-c952-11eb-986e-1ba1aa155dd2.png)
![Screenshot_20210609-183755](https://user-images.githubusercontent.com/6735650/121395038-45c56e00-c952-11eb-9324-2824c76b6c04.png)
![Screenshot_20210609-183803](https://user-images.githubusercontent.com/6735650/121395039-465e0480-c952-11eb-924d-3bd4d7953319.png)
![Screenshot_20210609-183809](https://user-images.githubusercontent.com/6735650/121395041-46f69b00-c952-11eb-860f-c923671c83be.png)
![Screenshot_20210609-183831](https://user-images.githubusercontent.com/6735650/121395043-478f3180-c952-11eb-8613-c407d1c33e80.png)
![Screenshot_20210609-183844](https://user-images.githubusercontent.com/6735650/121395046-4827c800-c952-11eb-94bc-f775cd16c730.png)
![Screenshot_20210609-183857](https://user-images.githubusercontent.com/6735650/121395048-48c05e80-c952-11eb-9f4e-62ac0b6019ff.png)
![Screenshot_20210609-183903](https://user-images.githubusercontent.com/6735650/121395050-4958f500-c952-11eb-8e03-29fd8c27ff2c.png)
![Screenshot_20210609-183911](https://user-images.githubusercontent.com/6735650/121395053-49f18b80-c952-11eb-8580-2b6c53228b60.png)

> I'm happy to accept code that adds an action where you can toggle the mode if a message is sent as multipart.

I somehow expected that this would be the response due the low priority label set :D

**Comment by ChristophWurst** [comment]
I think those requests are from the avatar logic. We use favicons as avatars if there is no contact with a picture in your address book.
